frontend/udp/frontend.go

```diff
@@ -66,6 +67,16 @@ type Frontend struct {
 
 // NewFrontend allocates a new instance of a Frontend.
 func NewFrontend(logic frontend.TrackerLogic, cfg Config) *Frontend {
+	if cfg.PrivateKey == "" {
```

Comment:
Can you add a log message for this? The user should know that this makes old connection IDs invalid.